Scoop/Controller/Ride Posting/PostRideLocationViewController.swift

```diff
@@ -118,3 +145,48 @@ class PostRideLocationViewController: UIViewController {
         }
     }
 }
+
+// MARK: - UITextFieldDelegate
+extension PostRideLocationViewController: UITextFieldDelegate {
+    
+    func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
+        return false
+    }
+    
```

Comment:
unnecessary spacing

Reply:
We actualy want to add one white space and below the protocol functions in an extension